.github/workflows/test.yml

```
@@ -0,0 +1,57 @@
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.10', '3.11', '3.12']
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v4
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)